apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -181,8 +181,10 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         state?.organization?.organizationType);
     const currentOrganization: OrganizationResponseInterface = useSelector((state: AppState) =>
         state.organization.organization);
+    const isFapiCompliantApp: boolean = initialValues?.isFAPIApplication;
 
     const [ isEncryptionEnabled, setEncryptionEnable ] = useState(false);
+    const [ isPublicClient, setPublicClient ] = useState(false);
```

Comment:
Types seems to be missing in the existing code as well. Lets atleast add to the new code block.
```suggestion
    const [ isPublicClient, setPublicClient ] = useState<boolean>(false);
```

Reply:
Applied with af36774